### PR TITLE
ARC 2044 - Create page modules to show in the connect sidebar at all times

### PR DESCRIPTION
**Pull request by krazziekay**
**Whats in it?**
- Added a web section which links to the Introduction page.
- Added a sidebar partial within the app.
- Added a header back to nav partial.

**Video**

https://user-images.githubusercontent.com/13076255/226770452-c890b106-3310-4594-ad0f-309b0731ea44.mov

### CODE REVIEW

File: src/views/layout.squirrelly [hunk 1]
```diff
@@ -4,8 +4,10 @@
     <title>{{ it.title }}</title>
     <meta charset="UTF-8">
     <link rel="stylesheet" href="https://unpkg.com/@atlaskit/css-reset/dist/bundle.css" media="all">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/aui/9.5.1/aui/aui-prototyping.min.css" />
```
Comment (rachellerathbone) [inline src/views/layout.squirrelly:7]: I think we should add a comment with link to AUI documentation + a mention to remove this if they don't want to use this styling.

File: static/css/back-nav.css [hunk 2]
```diff
@@ -0,0 +1,16 @@
+.acn-header {
+    cursor: pointer;
+    display: flex;
+    align-items: center;
+    color: #5E6C84;
+    font-size: 0.9rem;
+    margin-bottom: 24px;
```
Comment (rachellerathbone) [inline static/css/back-nav.css:7]: We started off using relative units for padding and margins. Just looked at the main.css file and realised our units are starting to get a little messy so should probably pick something and stick with. But that said, I wouldn't recommend using px for margin/padding. I know we're not really concerned with responsiveness (mainly in regards to mobile) but people will use different monitor/screen sizes so we shouldn't go with a fixed measurement like px.

File: static/css/sidebar.css [hunk 3]
```diff
@@ -0,0 +1,22 @@
+.acn-sidebar {
+    list-style: none;
+    padding: 1.6em 1.5em;
+    width: 50%;
+    max-width: 220px;
+    border-right: 1px solid #091e4214;
```
Comment (rachellerathbone) [inline static/css/sidebar.css:6]: Hex color codes have 6 characters. Why are there 8 here?

File: static/css/back-nav.css (hunk 2 above)
Comment (krazziekay) [reply static/css/back-nav.css]: Replaced px by em  for all margin and paddings.

File: src/views/layout.squirrelly (hunk 1 above)
Comment (krazziekay) [reply src/views/layout.squirrelly:8]: Added a comment

File: static/css/sidebar.css (hunk 3 above)
Comment (krazziekay) [reply static/css/sidebar.css]: Updated by a 6 character hex color code

**Review by rachellerathbone — APPROVED** [review]

**Pull request merged**